src/tools/pub/cmd/index.md

```diff
@@ -35,21 +35,21 @@ See [Troubleshooting Pub](/tools/pub/troubleshoot).
 
 Pub's commands fall into the following categories:
 
-* [Managing apps](#managing-apps)
+* [Managing package dependencies](#managing-apps)
 * [Running and serving apps](#running-and-serving-apps)
 * [Building
   and deploying apps and packages](#building-and-deploying-apps-and-packages)
 
-## Managing apps
+## Managing package dependencies {#managing-apps}
```

Comment:
Small suggestion: if you want to keep the old fragment id for backwards compatibility, I'd suggest having it in addition to the id generated by default from the section title: e.g.
```markdown
<a id="managing-apps"></a>
## Managing package dependencies
```

Reply:
That seems to be what markdown does (if you look at the generated code), although I suppose it wouldn't hurt to be explicit about it.

Reply:
If I am not mistaken, kramdown will use the id when specified via `{#...}`.

The other id that we're seeing is added by the toc script (which we might be changing soonish).

Reply:
Done. Thanks!